libcalico-go/lib/ipam/ipam.go

```diff
@@ -949,7 +949,17 @@ func (c ipamClient) AssignIP(ctx context.Context, args AssignIPArgs) error {
 
 // ReleaseIPs releases any of the given IP addresses that are currently assigned,
 // so that they are available to be used in another assignment.
-func (c ipamClient) ReleaseIPs(ctx context.Context, ips []net.IP) ([]net.IP, error) {
+func (c ipamClient) ReleaseIPs(ctx context.Context, ips ...ReleaseOptions) ([]net.IP, error) {
+	for i := 0; i < len(ips); i++ {
+		// Validate the input.
+		if ips[i].Address == "" {
+			return nil, fmt.Errorf("No IP address specified")
```

Comment:
Should we log more information than this? Granted, it doesn't make much sense for any of the elements of the `ReleaseOptions` to not have an address, but might be good to see if Handle information or something is passed in just in case this gets called in some funny use case.